### PR TITLE
Feature/rachelsophia/testimonials

### LINKED ISSUE #13: Testimonials Page
Description: Develop the Our Testimonials Page.

Task Type: Frontend, Backend
[Figma Here](https://www.figma.com/file/iMQp72nNAT8SsxOoiFgGSC/4FLOT-Design-Files?type=design&node-id=343-1570&mode=design&t=1PytETeKYKgjTzwC-0)

Frontend
* Background Image
* Testimonials Card component (image, quote, description)
* Past Events component (image, title, description)

Backend
* GET request for all testimonials
* GET request for past events (hold off on this until upcoming events is set up)

### PR DESCRIPTION
**Pull request by syz16**
## Tracking Info

Resolves #13 

## Changes

We created the Testimonials page. We implemented backend for the testimonials as well as for the events. For the frontend, we made separate cards to display both the testimonial and event. We formatted the CSS to match the Figma design.

## Testing

We used postman for the backend to ensure all API POST and GET requests were correct. For the frontend, we visually confirmed that it looks like the Figma, and we resize the window the cards shift/wrap as expected.

## Confirmation of Change
Screenshot 2024-02-12 at 4 13 30 PM

Screenshot 2024-02-12 at 4 13 43 PM

### CODE REVIEW

File: frontend/src/app/impact/page.css
Comment (christen03) [inline frontend/src/app/impact/page.css:1]: can we do this in tailwind instead so we don't have unneeded css files?

Comment (christen03) [reply frontend/src/app/impact/page.css:1]: nvm probably fine because u are doing it for p/h1 and they r commonly used

**Review by christen03 — APPROVED** [review]
looks good! love the use of validator :)

**Review by jennymar — APPROVED** [review]
LGTM! Thanks so much for making the backend routing changes and frontend fixes. I cleaned up the CSS so that TestimonialCard can be used for both the quotes and past event cards to avoid overlapping code! great work!

**Pull request merged**